### ISSUE TITLE
Erroneously removed CSS

### ISSUE THREAD

**Issue by ad-si** [svg/svgo#952]
I'm having problems with erroneously removed CSS since the update from 0.7.2 to 1.0.x.
Only default plugins are enabled (I misunderstood the https://github.com/svg/svgo/issues/946 issue).

It seems to be caused by a CSS hierarchy it doesn't resolve correctly. Can't really pin it down.
Just moving the CSS rule up one level to the parent element fixed the problem.

**Comment by GreLI** [comment]
Could you provide a sample?

**Comment by ad-si** [comment]
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
  <style>
    #bot {
      stroke: hsl(0, 0%, 30%);
      fill: hsl(0, 0%, 30%);
    }

    #bot .closed-eye {
      fill: none;
      stroke-linecap: round;
      stroke-width: 2;
    }
  </style>

  <g id="bot" transform="translate(3, 2)">
    <g class="head">
      <g class="features" transform="translate(1, -2)">
        <g class="eyes" transform="translate(4, 15)">
          <path class="closed-eye" d="M0,0 c1,-1.3 3,-1.3 4,0" />
          <path class="closed-eye" d="M12,0 c1,-1.3 3,-1.3 4,0" />
        </g>
      </g>
    </g>
  </g>
</svg>
```
The styling for `.closed-eye` is discarded

**Comment by GreLI** [comment]
Yeah, seems like a bug in `inlineStyles` plugin. You can disable it as workaround.
